backend/src/Altinn.AccessManagement.UI/Altinn.AccessManagement.UI.Core/Services/ResourceService.cs

```diff
@@ -57,9 +61,20 @@ public async Task<PaginatedList<ServiceResourceFE>> GetPaginatedSearchResults(st
                 List<ServiceResource> resourceList = resources.FindAll(r => r.ResourceType != ResourceType.MaskinportenSchema && r.ResourceType != ResourceType.SystemResource && r.Delegable && r.Visible);
                 List<ServiceResourceFE> resourcesFE = MapResourceToFrontendModel(resourceList, languageCode);
 
-                List<ServiceResourceFE> filteredresources = FilterResourceList(resourcesFE, resourceOwnerFilters);
-                List<ServiceResourceFE> searchResults = SearchInResourceList(filteredresources, searchString);
-                return PaginationUtils.GetListPage(searchResults, page, resultsPerPage);
+                bool displayPopularServicesOnly = _logicFlags.DisplayPopularSingleRightsServices;
+                if (string.IsNullOrEmpty(searchString) && (resourceOwnerFilters == null || resourceOwnerFilters.Length == 0) && displayPopularServicesOnly)
+                {
+                    // Return a selection of popular services
+                    List<ServiceResourceFE> popularResources = FilterOutPopularResources(resourcesFE);
+                    return PaginationUtils.GetListPage(popularResources, page, resultsPerPage);
```

Comment:
I'm not sure how to write this into a test when its based on environment variables.. Any suggestions?
Or isn't it needed do you think?

Reply:
Hm. I'm not sure how we could do this. Maybe we can override the value in "Logic-Flags" somewhere, but I'm not entirely sure how we could do that. It's probably not the most important thing to test, so I don't think we need to worry about it at the moment.

Reply:
I'm thinking the same thing.
If this becomes a recurring problem with feature-flags and the likes, we can take a look at that point